robotino_description/urdf/base.urdf.xacro

```diff
@@ -1,116 +1,226 @@
 <?xml version="1.0"?>
 <root xmlns:xacro="http://www.ros.org/wiki/xacro">
 
-  <xacro:include filename="$(find robotino_description)/urdf/gazebo.urdf.xacro" />
+  <xacro:include filename="$(find robotino_description)/urdf/base.gazebo.xacro" />
+  <xacro:include filename="$(find robotino_description)/urdf/base.transmission.xacro" />
 
-  <xacro:macro name="base" params="name">
-  
-    <!--link name="${name}_footprint">
+  <xacro:macro name="base" params="name has_8dof">
+
+  <link name="origin" />
+
+  <joint name="origin_joint" type="fixed">
```

Comment:
I am not sure if it is right, theoretically the festo driver should update this transformation